css/style.css

```diff
@@ -0,0 +1,474 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+@import url('../fonts/Cocogoose\ Pro-trial.ttf');
+
+:root {
+  --color-primary: #272626;
+  --color-secondary: #ec5242;
+  --color-tertiary: #d3d3d3;
+  --color-cards: #d3d3d324;
+  --header-font: 'Cocogoose';
+  --body-font: 'Lato', sans-serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+.tumbnail1 {
+  display: flex;
+  width: 50px;
+  height: 3px;
+  margin: 5px auto 10px auto;
+  border: solid 2px var(--color-primary);
+}
+
+.tokio {
+  width: 80vw;
+  height: 80vw;
+  text-align: center;
+  background-image: url('../images/tokio.png');
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-color: #ec5342cb;
+  background-blend-mode: overlay;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.dubai {
+  width: 80vw;
+  height: 80vw;
+  text-align: center;
+  background-image: url('../images/dubai.png');
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-color: #ec5342d5;
+  background-blend-mode: overlay;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: var(--body-font);
+  font-size: 16px;
+  line-height: 1.5;
+  color: var(--color-primary);
+  background-color: var(--color-tertiary);
+  list-style: none;
+  height: 80vh;
+  width: 100%;
+  overflow-x: hidden;
+}
+
+.topnav {
+  display: none;
+}
+
+header {
+  font-family: var(--header-font);
+  background-color: var(--color-primary);
+  list-style: none;
+  text-decoration: none;
+  width: 100%;
+}
+
+nav > ul > li {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+  text-decoration: none !important;
```

Comment:
- [ ] [OPTIONAL] Just for future reference try avoiding using `!important`. with the amount of CSS you have you really shouldn't need this.

Reply:
I'm well aware that is not a good practice use the ***! important*** technique, but I wasn't able to turn of the markers... anyway I'll take a note.